apps/video/index.html

```diff
@@ -33,9 +33,11 @@
 
     <!-- Web Components -->
     <script defer src="shared/elements/config.js"></script>
+    <script defer src="shared/js/component_utils.js"></script>
```

Comment:
Nit: not related to your change so much, but for readability I think it makes sense to keep a logical ordering if possible. Can the 'elements' and 'js' scripts be grouped together?